File: mario_pyramid/JS/mario.js
console.log("Mario Pyramid With JS!");



// Prints a Pyramid given a size
const makePyramid = (size) => {

	// Rows
	for(let row = "#"; row.length <= size; row+='#') 
		console.log(row);
}

makePyramid(10);



//Prints Reverse Pyramid
const makeReversePyramid = (size) => {

	// Rows
	for(let row = "#", spaces; row.length <= size; row+='#') {
		// Spaces
		for(spaces = ""; spaces.length <= (size - row.length); spaces+=' ');
			
		console.log(spaces + row);
	}
}


makeReversePyramid(10);

File: mario_pyramid/C++/mario.cpp
#include <iostream>

// Prints Mario Pyramid in given size.
void marioPyramid(int size);

int main() {
	std::cout << "Mario Pyramid in C++!" << std::endl;


	marioPyramid(10);

};

void marioPyramid(int size) {
	// Print rows
	for (int i = 0; i < size; i ++) {
		// Print columns
		for (int j = 0; j < i+1; j ++)
			std::cout << '#';
	std::cout << std::endl;
	}
};